src/database/DataTypes/Requisition.js

```diff
@@ -497,6 +506,7 @@ Requisition.schema = {
     linkedTransaction: { type: 'Transaction', optional: true },
     program: { type: 'MasterList', optional: true },
     period: { type: 'Period', optional: true },
+    linkedRequisition: { type: 'string', optional: true },
```

Comment:
This won't work for historic requisitions, you would need to create migration code to send a message to the server saying the mobile site has upgraded, then have the server send all customers requisitions back to all mobile sites so we can save the id of the linked requisition.

An easier solution is to use `linkedTransaction`. All customer requisitions which have come from the server will have a `linkedTransaction` on mobile, while manually created ones on the mobile site itself will not.

This means we won't need to create migration code, nor will we need to add this field